product/index.html

```diff
@@ -19,17 +19,18 @@
         />
         <link rel="stylesheet" href="productstyles.css" />
         <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="https://neo-ui.netlify.app/styles.css">
         
     </head>
     <body>
-        <nav class="navbar navbar-white bg-white">
-            <p>MyWebsite</p>
+        <nav class="navbar">
+            <p class="head">MyWebsite</p>
```

Comment:
header as class name can be more suitable and relevant here

Reply:
Can use `<h5>/ <h6>` instead